plugin/wavesurfer.minimap.js

```diff
@@ -94,7 +94,7 @@ WaveSurfer.Minimap = WaveSurfer.util.extend({}, WaveSurfer.Drawer, WaveSurfer.Dr
 
     bindWaveSurferEvents: function () {
         var my = this;
-        this.wavesurfer.on('ready', this.render.bind(this));
+        this.wavesurfer.on('ready', my.render());
```

Comment:
@piem what code are you using that it gives the error? 

@katspaugh without the change it wouldn't show the minimap on on ready event and the user is required to resize the window for it to work properly or call minimap.render(); right after.

````
    var minimap1 = wavesurfer.initMinimap({
        container: document.querySelector('#minimap'),
        height: 30,
        waveColor: '#ddd',
        progressColor: '#999',
        cursorColor: '#68A93D'
    });
    //without calling this it won't render on ready
    minimap1.render();
````


In the resize event it's calling my.render(); which makes it work when resizing the window. On the ready event it's not calling it so it won't render the minimap until you resize the window or call minimap.render() yourself after wavesurfer.initMinimap().

````
    var onResize = function () {
            if (prevWidth != my.wrapper.clientWidth) {
                prevWidth = my.wrapper.clientWidth;
                my.render();
                my.progress(my.wavesurfer.backend.getPlayedPercents());
            }
        };
````